app/models/roster.rb

```diff
@@ -69,4 +69,12 @@ def user_options
     nas = non_admins.order(:last_name).map { |na| [na.full_name, na.id] }
     { 'Admins' => as, 'Non-Admins' => nas }
   end
+
+  def check_for_open_dates_between(start_date, end_date)
```

Comment:
I personally am not a fan of `check_` style naming for methods that return data, especially if no "action" is being performed in accordance with the check. IMO it's a lot more ruby-like to just name the method according to what's returned, i.e.

```suggestion
  def uncovered_dates_between(start_date, end_date)
```

Just my opinion though